src/services/email.service.ts

```diff
@@ -0,0 +1,42 @@
+import * as SendGridMail from '@sendgrid/mail';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+SendGridMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+/* Send an email to one or more recipients */
+const _sendEmails = async (
+  emailInfoArr: SendGridMail.MailDataRequired[]
+): Promise<void> => {
+  const [
+    {
+      to,
+      dynamicTemplateData,
+      templateId,
+      cc,
+      bcc,
+      sendAt,
+      ...additionalFields
+    },
+    ...additionalEmailInfoObjs
+  ] = emailInfoArr;
+
+  try {
```

Comment:
skip try catch if you're just gonna throw; add todo to add logging when we have that setup